README.md

```diff
@@ -191,6 +210,9 @@ Other variables may or may not be useful.
 Run `cargo test` in the repository root.
 
 ## Parallel rust-analyzer builds
+TODO: revisit this whole section; idea: maybe it should be some sort of "common
```

Comment:
Yes, I don't know how many of us actually use this workflow (I don't) — is this still current? etc.

Reply:
Ended up removing TODOs in order to merge this as-is for now. Will try to revisit these later.